tools/serverpod_cli/lib/src/serverpod_packages_version_check/serverpod_packages_version_check.dart

```diff
@@ -24,7 +24,10 @@ List<SourceSpanException> performServerpodPackagesAndCliVersionCheck(
 ) {
   List<SourceSpanException> accumulatedWarnings = [];
 
-  var pubspecFiles = findPubspecsFiles(directory);
+  var pubspecFiles = findPubspecsFiles(
+    directory,
+    ignorePaths: ['vendor/serverpod'],
```

Comment:
I think it should probably exclude paths under both vendor directories and serverpod directories (not just vendor/serverpod). Also, this may not be working on Windows the the path delimiter.